packages/channels/botpress-channel-web/src/views/web/convo/index.jsx

```diff
@@ -103,15 +103,18 @@ export default class Convo extends React.Component {
 
     if (this.props.config.botAvatarUrl) {
       content = (
-        <div className={style.picture} style={{ backgroundImage: 'url(' + this.props.config.botAvatarUrl + ')' }} />
+        <div
+          className={`${style.picture} botpress__picture`}
+          style={{ backgroundImage: 'url(' + this.props.config.botAvatarUrl + ')' }}
+        />
       )
     }
 
     return (
-      <div className={style.avatar}>
-        <div className={style.square}>
+      <div className={`${style.avatar} botpress__avatar`}>
+        <div className={`${style.square} botpress__square`}>
```

Comment:
Not sure these are good public names. If they're only used here let's call them something like `botpress__avatar-container-outer` and `botpress__avatar-container-inner` (square and circle, respectively)